src/test/java/com/dawn/jat/illuminati/post/advice/PostExceptionAdviceTest.java

```diff
@@ -29,10 +31,5 @@ void exception_throwException_returnHttpPostSummaryNoContent() {
         ResponseEntity<ErrorResponse> rsp = advice.exceptionPostSummary(postSummaryException);
         assertEquals("Cannot find post summary", postSummaryException.getMessage());
         assertEquals(HttpStatus.OK, rsp.getStatusCode());
-
-        ErrorResponse errResp = new ErrorResponse(HttpStatus.NO_CONTENT, postSummaryException);
```

Comment:
Sorry :3, I made a mistake last time, this part is actually ok, but dont involve ErrorResponse errResp here, it's not neccessary. And the code make people confuse, the actual parameter is stand as second param `errResp.getError()`. So for a second, I thought you were testing with the errResp

The code should be in form `assertEquals(expect, actual)` and straightforward as bellow:
```
assertEquals("Cannot find post summary", rsp.getBody().getError());
assertEquals(false, rsp.getBody().getSuccess());
```

Reply:
Build Fail here ==> assertEquals("Cannot find post summary", rsp.getBody().getError());

Reply:
Updated. Mocking an object Map<String, Object> error instead of involve ErrorResponse errResp